### DISCUSSION TITLE
Does Iris shaders work with rubidium or embeddium

### DISCUSSION THREAD

**Discussion by Cfcsfsf** [Sinytra/Connector#810] [Q&A] [answered]
so i have problem with oculus so i wanted to use iris using sinytra connector

**Comment by kakashi1464** [comment]
What's problem with oculus?

**Reply by Cfcsfsf** [reply]
crashes always even tried to use older version of rubidium and oculus(aka oculus 1.6.4)

**Comment by u-u-x** [comment, marked as answer]
> crashes always even tried to use older version of rubidium and oculus(aka oculus 1.6.4)
I just made it work using embeddium instead of rubidium, and it works just fine! (its been a while so im not sure if youre still interested, but ill leave it here for others with the same question)
